lib/membrane_h264_plugin/parser/nalu.ex

```diff
@@ -8,16 +8,14 @@ defmodule Membrane.H264.Parser.NALu do
   A type defining the structure of a single NAL unit produced
   by the parser.
```

Comment:
Would be nice to provide descriptions for these fields. 

In particular, if possible, I would clarify `type` (`atom() -> :idr | :non_idr | etc.`)

Reply:
done